### PR TITLE
Igal/upload logs

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: src/ops/ops.go [hunk 1]
```diff
@@ -347,3 +348,12 @@ func (o *ops) GetMCSLogs() (string, error) {
 
 	return string(logs), nil
 }
+
+func (o *ops) UploadInstallationLogs() (string, error) {
+	command := "podman"
+	args := []string{"run", "--rm", "--privileged", "-v", "/run/systemd/journal/socket:/run/systemd/journal/socket",
+		"-v", "/var/log:/var/log", "docker.io/tsorya/logs_sender:latest", "logs_sender", "-tag", "agent", "-tag", "installer",
```
Comment (romfreiman) [inline src/ops/ops.go:355]: nice image :)

Comment (tsorya) [inline src/ops/ops.go:355]: fix it

Comment (romfreiman) [reply src/ops/ops.go:355]: rega, so we have to build this one as well? where?

Comment (tsorya) [reply src/ops/ops.go]: nope it was bug

**Pull request merged**